mindjunkies/forums/views.py

```
@@ -99,3 +105,24 @@
+            topic.save()
+        else:
+            # User is reacting
+            topic.reaction.add(request.user)
+            # Update like_count
+            # topic.like_count += 1
+            topic.save()
+            
+        return HttpResponseRedirect(
+            f"{reverse('forum_home', kwargs={'course_slug': course_slug})}"
+        )
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)